Rubeus/Commands/Golden.cs

```diff
@@ -383,6 +384,19 @@ public void Execute(Dictionary<string, string> arguments)
                 Console.WriteLine("\r\n[X] You must supply a [/des|/rc4|/aes128|/aes256] hash!\r\n");
                 return;
             }
+            else
+            {
+                try
+                {
+                    hashBytes = Helpers.StringToByteArray(hash);
```

Comment:
See previous comments regarding propagating the exception and catching higher up in Main